controller/jiminy/src/seeds.rs

```diff
@@ -0,0 +1,7 @@
+use inf1_ctl_core::{keys::POOL_STATE_BUMP, pda::POOL_STATE_SEED};
+use jiminy_pda::{PdaSeed, PdaSigner};
+
+pub const POOL_SEED_SIGNERS: PdaSigner = PdaSigner::new(&[
```

Comment:
nit: this is a single signer
```suggestion
pub const POOL_SEED_SIGNER: PdaSigner = PdaSigner::new(&[
```